### PR TITLE
minor doc fixes for #93

### PR DESCRIPTION
**Pull request by dacoex**

### CODE REVIEW

File: docs/sphinx/source/whatsnew/v0.3.0.txt [hunk 1]
```diff
@@ -49,9 +52,12 @@ Bug fixes
   (:issue:`105`)
 * Fixed the metadata key specification in documentation of the
   readtmy2 function.
+* Fixes the import of tkinter (:issue:`112`)
```
Comment (wholmgren) [inline docs/sphinx/source/whatsnew/v0.3.0.txt]: on Python 3

**Comment by dacoex** [comment]
@wholmgren from my side no more additions to v.0.3

Thanks for the effort you put into all this: refratoring, discussions, etc. It's really appreciated.

**Comment by wholmgren** [comment]
@dacoex Did you mean to close this?

**Comment by dacoex** [comment]
Yes, in the that comment is solved and code read to merge

**Comment by wholmgren** [comment]
Thanks. 

In the future, just comment that you fixed it. I can't merge a closed PR.

**Pull request merged**